### ISSUE TITLE
Firefox port and other small suggestions 

### ISSUE THREAD

**Issue by Jeusto** [Ademking/BetterViewer#11]
Hey, I don't know if you've looked into porting the extension to Firefox but I've tried a few things and here's what I've noticed: 
The extension mostly seems compatible with both browsers since Firefox supports "browser.*" namespace for APIs as well as "chrome.\*".  
The first problem is Firefox requires an addon id in the "manifest.json" in order to use "[storage.sync](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/storage/sync)". This can simply be fixed by adding something like :
```json
"browser_specific_settings": {
  "gecko": {
    "id": "51846946-47ad-4ea1-bac3-ed242f738cf1",
  }
```
This throws a warning called "unrecognized manifest key" in Chrome, it's just a warning so not a big deal but can be fixed by having separate "manifest.json" files for both browsers. 

The second main problem is "Crop image" and "Extract text" are not working, here's how it looks like. They both use "cropper.js" I believe, so that might be where the problem comes from ?

![Screenshot-2022-01-25-16-59](https://user-images.githubusercontent.com/65134804/151012965-a69f6211-0820-4ce6-a34d-bf609b0c2c81.png)
 
I have other small suggestions like for example: moving the windows or toasters at the top by default because when the browser window is not wide enough, they tend to overlap with the toolbar which can be annoying but these are small changes so I can make a pull request if you're interested :

![Screenshot-2022-01-25-17-12](https://user-images.githubusercontent.com/65134804/151015043-74f0886c-858d-4d86-96d4-b754ec23e356.png)

**Comment by Ademking** [comment]
Hello @Jeusto,
Sorry for the late reply! and thank you so much for your suggestions!

It's working now on Firefox 🔥 🦊 
I've fixed the crop problem (I've changed "croppr.js" library)
And I've added some customization like toast position, toolbar position, persistent theme, etc...

![image](https://user-images.githubusercontent.com/12462188/153190037-b52396f4-8817-4ccc-a80a-d65901f23782.png)

![image](https://user-images.githubusercontent.com/12462188/153190090-839add05-896e-4b6d-aae2-3b6877f33304.png)

![image](https://user-images.githubusercontent.com/12462188/153190160-d015faac-a6f4-49bc-afe9-c7a595837f87.png)

Thank you so much!
